runelite-client/src/main/java/net/runelite/client/plugins/chatnotifications/ChatNotificationsConfig.java

```diff
@@ -28,34 +28,59 @@
 import net.runelite.client.config.Config;
 import net.runelite.client.config.ConfigGroup;
 import net.runelite.client.config.ConfigItem;
+import net.runelite.client.config.ConfigSection;
 
 @ConfigGroup("chatnotification")
 public interface ChatNotificationsConfig extends Config
 {
+	@ConfigSection(
+			name = "Highlight Lists",
+			description = "Custom single Word, and Regex filter lists",
+			position = 0,
+			closedByDefault = true
+	)
+	String highlightLists = "highlightLists";
+
 	@ConfigItem(
-		position = 0,
-		keyName = "highlightOwnName",
-		name = "Highlight own name",
-		description = "Highlights any instance of your username in chat"
+			position = 1,
+			keyName = "highlightWordsString",
+			name = "Highlight words",
+			description = "Highlights the following words in chat",
+			section = highlightLists
 	)
-	default boolean highlightOwnName()
+	default String highlightWordsString()
 	{
-		return true;
+		return "";
 	}
 
 	@ConfigItem(
-		position = 1,
-		keyName = "highlightWordsString",
-		name = "Highlight words",
-		description = "Highlights the following words in chat"
+			position = 2,
+			keyName = "highlightRegexString",
+			name = "Highlight Regex",
+			description = "Highlights the following regular expressions in chat, one per line",
+			section = highlightLists
 	)
-	default String highlightWordsString()
+	default String highlightRegexString()
 	{
 		return "";
 	}
 
+
 	@ConfigItem(
-		position = 2,
+		position = 3,
```

Comment:
Ok, but is there any disadvantage to doing it? I've just mimicked the way chatFilterConfig was set up.

Reply:
No, but there's no advantage to changing these. Sorting is done both at the config level, and within config sections, so this:
```
0: section
  0: section-item
  1: section-item
  2: section-item
1: top-level-item
2: top-level-item
```
will display exactly the same as this:
```
0: section
  1: section-item
  2: section-item
  3: section-item
4: top-level-item
5: top-level-item
```
As-is, this PR unnecessarily changes existing config positions to have ordering like the second example when it will make no difference.

Reply:
Alright, I'll switch those back tomorrow morning.